src/themes/teams/withProcessedIcons.ts

```diff
@@ -1,29 +1,38 @@
 import { ThemeInput, ThemeIcons, ThemeIconSpec, SvgIconSpec } from '../types'
 
 import { default as svgIconsAndStyles } from './components/Icon/svg/ProcessedIcons'
-import { TeamsProcessedSvgIconSpec, SvgIconSpecWithStyles } from './components/Icon/svg/types'
+import { TeamsProcessedSvgIconSpec, SvgIconSpecWithMetaData } from './components/Icon/svg/types'
 
 type ThemeProcessedIconSpec = ThemeIconSpec &
   { [K in keyof TeamsProcessedSvgIconSpec]?: TeamsProcessedSvgIconSpec[K] }
 
-const declareSvg = (svgIcon: SvgIconSpec, exportedAs?: string): ThemeProcessedIconSpec => ({
+const declareSvg = (
+  svgIcon: SvgIconSpec,
+  rotateInRtl,
+  exportedAs?: string,
+): ThemeProcessedIconSpec => ({
   isSvg: true,
   icon: svgIcon,
+  rotateInRtl,
   exportedAs,
 })
 
 const processedIcons: ThemeIcons = Object.keys(svgIconsAndStyles as {
   [iconName: string]: TeamsProcessedSvgIconSpec
 }).reduce<ThemeIcons>((accIcons, iconName) => {
-  const iconAndMaybeStyles = svgIconsAndStyles[iconName]
+  const iconAndMaybeMetaData = svgIconsAndStyles[iconName]
 
-  const icon: SvgIconSpec = (iconAndMaybeStyles as any).styles
-    ? (iconAndMaybeStyles as SvgIconSpecWithStyles).icon
-    : (iconAndMaybeStyles as SvgIconSpec)
+  const icon: SvgIconSpec = (iconAndMaybeMetaData as any).styles
+    ? (iconAndMaybeMetaData as SvgIconSpecWithMetaData).icon
+    : (iconAndMaybeMetaData as SvgIconSpec)
+
+  const rotateInRtl = (iconAndMaybeMetaData as any).rotateInRtl
+    ? (iconAndMaybeMetaData as any).rotateInRtl
```

Comment:
do we need this ternary operator? I would expect it being safe to just directly provide the value of `(iconAndMaybeMetaData as any).rotateInRtl` as `rotateInRtl`

Reply:
Have reverted these changes.